File: problems/problems_3591/Solution.java
package problems.problems_3591;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public boolean checkPrimeFrequency(int[] nums) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        int[] nums = jsonArrayToIntArray(inputJsonValues[0]);
        return JSON.toJSON(checkPrimeFrequency(nums));
    }
}

File: problems/problems_3591/solution.go
package problem3591

import (
	"encoding/json"
	"log"
	"strings"
)

func checkPrimeFrequency(nums []int) bool {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var nums []int

	if err := json.Unmarshal([]byte(inputValues[0]), &nums); err != nil {
		log.Fatal(err)
	}

	return checkPrimeFrequency(nums)
}
